### PR TITLE
Add ability to run external AP and switch modes, update readme 

### PR DESCRIPTION
**Pull request by danparisd**
This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/btt-exchanger/8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,4 +25,21 @@ or
 
 
 ## About the install.sh script
-The script adds the docker repo, installs dependencies, adds a cron job to start tE, pulls the needed docker images, and eventually reboots the machine to start tE via the cron job
+- The script adds the docker repo, installs dependencies, adds a cron job to start tE, pulls the needed docker images, and eventually reboots the machine to start tE via the cron job.
```
Comment (mXaln) [inline README.md]: BTT Exchanger? Line 28

File: README.md [hunk 2]
```diff
@@ -23,4 +25,21 @@ or
...
+
+- There are two prompts to respond to in the install script. The first asks whether you want to install a Wifi Access Point on the server (NUC) itself or if you want to use an external Access Point like the TP-Link mentioned above. The second prompt is to unlpug the network cable for reboot. The install sets up a DHCP server, so unplugging the network cable before rebooting and starting the DHCP server is highly recommended. 
+
+## Use
+1. After a reboot, either connect to the TP-Link Wireless Netowrk if using the TP-Link router or connect to the TranslationExchange_##### Wireless network if you are running the Wireless Access Point on the server. The setup script for using an external access point configures the DHCP and DNS servers in a way that requires the TP-Link nano router to be configured in Access Point mode. See the TP-Link documentation for how to change this config. Feel free to adjust its config such as admin password, SSID, and wireless password as well.
```
Comment (mXaln) [inline README.md]: BTT Exchanger? Line 33

File: install.sh [hunk 3]
```diff
@@ -43,10 +63,12 @@ rm -rf clients/
 unzip clients.zip
 
 echo -e "${COLOR}----------| Downloading TranslationRecorder|----------${NC}"
+
 URL=$(curl 'https://api.github.com/repos/wycliffeassociates/translationrecorder/releases?per_page=1' | jq -r '.[0] | .assets[].browser_download_url')
```
Comment (mXaln) [inline install.sh]: Should now download from https://api.github.com/repos/Bible-Translation-Tools/BTT-Recorder/releases

File: install.sh [hunk 4]
```diff
@@ -43,10 +63,12 @@ rm -rf clients/
...
 curl -L $URL --output clients/translationRecorder.apk
```
Comment (mXaln) [inline install.sh]: bttRecorder.apk

**Review by mXaln — CHANGES_REQUESTED** [review]
Everything looks good to me, just some branding issues...

File: README.md (hunk 2 above)
Comment (danparisd) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (danparisd) [reply README.md]: done

File: install.sh (hunk 3 above)
Comment (danparisd) [reply install.sh]: done

File: install.sh (hunk 4 above)
Comment (danparisd) [reply install.sh]: done

File: README.md [hunk 5]
```diff
@@ -23,4 +25,21 @@ or
 
 
 ## About the install.sh script
-The script adds the docker repo, installs dependencies, adds a cron job to start tE, pulls the needed docker images, and eventually reboots the machine to start tE via the cron job
+- The script adds the docker repo, installs dependencies, adds a cron job to start tE, pulls the needed docker images, and eventually reboots the machine to start BTT-Exchanger via the cron job.
```
Comment (mXaln) [inline README.md]: Rename tE

Comment (danparisd) [reply README.md]: done

**Review by mXaln — CHANGES_REQUESTED** [review]
Just one last branding change

**Review by mXaln — APPROVED** [review]

**Pull request merged**